scripts/build/tests.js

```diff
@@ -124,8 +123,8 @@ async function runConsoleTests(runJs, defaultReporter, runInParallel, watchMode,
             // finally, do a sanity check and build the compiler with the built version of itself
             log.info("Starting sanity check build...");
             // Cleanup everything except lint rules (we'll need those later and would rather not waste time rebuilding them)
-            await exec("gulp", ["clean-tsc", "clean-services", "clean-tsserver", "clean-lssl", "clean-tests"], { cancelToken });
-            const { exitCode } = await exec("gulp", ["local", "--lkg=false"], { cancelToken });
+            await exec("node", [process.argv[1], "clean-tsc", "clean-services", "clean-tsserver", "clean-lssl", "clean-tests"], { cancelToken });

```

Comment:
what is `process.argv[1]` here? Is it `gulp`? How does that avoid the not found error? Is it sometimes the full path to gulp? Does the user have to provide the full path?

Reply:
It's the [full path to the main JavaScript file being executed](https://nodejs.org/api/process.html#process_process_argv). I double checked and I don't need to provide the full path: `process.argv[1]` is the full path when I provide a relative path, e.g.
```Shell
> CI=true relative/gulp runtests-parallel
```
or
```Shell
> CI=true node relative/gulp runtests-parallel
```